mono/mini/Makefile.am.in

```diff
@@ -963,5 +963,6 @@ tags:
 if HAS_EXTENSION_MODULE
 else
 Makefile.am: Makefile.am.in
-	cp $< $@
+	echo "##################### Generated from Makefile.am.in, do not modify ##########################" > $@
```

Comment:
Even better would be every few lines, so people won't miss it, which I understand isn't easy, can't interrupt the code at arbitrary places.

Reply:
That would be unusually complicated due to make syntax, wouldn't it?

Reply:
Agreed.